TTD/TTD.hpp

```diff
@@ -437,7 +489,84 @@ namespace TTD {
 		GuestAddress GetPebAddress();
 		Cursor NewCursor();
 		unsigned __int64 GetModuleCount();
-		struct TTD_Replay_Module* GetModuleList();
+		const TTD_Replay_Module* GetModuleList();
+
+		/*!
+		 * \brief	Number of loaded module event (on load event)
+		 * \return	Number of loaded module event (on load event)
+		 */
+		unsigned __int64 GetModuleLoadedEventCount();
+
+		/*!
+		 * \brief	Array of loaded module event
+		 * \warning	use GetModuleLoadEventCount to know the number of event
+		 * \return	pointer to the first event
+		 */
+		const TTD_Replay_ModuleLoadedEvent* GetModuleLoadedEventList();
+
+		/*!
+		 * \brief	More c++ interface using vector
+		 *			list of module loaded event
+		 * \return	list of TTD_Replay_ModuleLoadedEvent
+		 */
+		const std::vector<TTD_Replay_ModuleLoadedEvent> GetModuleLoadedEvent();
```

Comment:
```suggestion
		const std::vector<TTD_Replay_ModuleLoadedEvent> GetModuleLoadedEvents();
```
Should we add a `s` to all these interfaces, to be more meaningfull?